tests/system/test_client.py

```diff
@@ -123,7 +125,7 @@ def _has_rows(result):
 
 
 def _make_dataset_id(prefix):
-    return "%s%s" % (prefix, unique_resource_id())
+    return "python_bigquery_tests_system_%s%s" % (prefix, unique_resource_id())
```

Comment:
Towards https://github.com/googleapis/python-bigquery/issues/590

Reply:
(nit) For consistency, should we change this to an f-string?
```suggestion
    return "python_bigquery_tests_system_{prefix}{unique_resource_id()}"
```